libvast/test/format/writer.cpp

```diff
@@ -45,8 +45,8 @@ auto first_zeek_conn_log_line = R"__({"ts": 1258531221.486539, "uid": "Pii6cUUq1
 template <class Writer>
 std::vector<std::string> generate(const std::vector<event>& xs) {
   std::string str;
-  auto sb = new caf::containerbuf<std::string>{str};
```

Comment:
Wow, a naked `new`...and that in 2019!

Reply:
Technically, a naked `new` from 2016: https://github.com/vast-io/vast/commit/1b479ee1220ae432523d6a03e86483434515712c